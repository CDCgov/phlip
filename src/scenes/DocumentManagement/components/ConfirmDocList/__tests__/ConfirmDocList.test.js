import React from 'react'
import { shallow } from 'enzyme'
import { ConfirmDocList } from '../index'

const props = {
  documents: [{ name: 'Doc', _id: '1' }], docCount:1
}

describe('DocumentManagement - DocList', () => {
  test('should render correctly', () => {
    expect(shallow(<ConfirmDocList {...props} />)).toMatchSnapshot()
  })
})