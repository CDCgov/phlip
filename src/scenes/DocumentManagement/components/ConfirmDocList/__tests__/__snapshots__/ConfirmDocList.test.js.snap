// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentManagement - DocList should render correctly 1`] = `
<FlexGrid
  align="stretch"
  circular={false}
  container={true}
  flex={true}
  justify="stretch"
  padding={0}
  raised={false}
  style={
    Object {
      "overflow": "hidden",
    }
  }
  type="column"
>
  <FlexGrid
    align="stretch"
    circular={false}
    container={false}
    flex={false}
    justify="stretch"
    padding={0}
    raised={false}
    style={Object {}}
    type="column"
  >
    <WithStyles(Typography)
      align="center"
    >
      Number of Documents Selected: 
      1
    </WithStyles(Typography)>
  </FlexGrid>
</FlexGrid>
`;
